azerothcore/azerothcore-wotlk
# feat(Core): Allow disable points of interest for quests

## Changes Proposed:
-  Add a config option to be able to disable points of interest for quests shown on the map

It won't affect `!` or `?` shown on the minimap, only the markers shown on the regular map.

It's intended for progression purposes as points of interest were added in in the middle of Wrath of the Lich King.

They are enabled by default.

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Enabled the config option, which it is by default, and accepted quests and saw the markers on the map
- Disabled the config option and accepted quests and didn't see any markers on the map


## How to Test the Changes:


1. Have the config option set to 1 (enabled) and accept some quests and make sure the markers are shown on your map
2. Have the config option set to 0 (disabled) and accept some quests and make sure there are no markers shown on the map

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
